--- source/common/unicode/uconfig.h	2008-07-01 08:41:08.000000000 +0700
+++ source/common/unicode/uconfig.h	2008-08-21 16:22:57.000000000 +0700
@@ -59,7 +59,7 @@
  * @stable ICU 2.4
  */
 #ifndef UCONFIG_ONLY_COLLATION
-#   define UCONFIG_ONLY_COLLATION 0
+#   define UCONFIG_ONLY_COLLATION 1
 #endif

 #if UCONFIG_ONLY_COLLATION
@@ -128,7 +128,7 @@
  * @stable ICU 2.4
  */
 #ifndef UCONFIG_NO_LEGACY_CONVERSION
-#   define UCONFIG_NO_LEGACY_CONVERSION 0
+#   define UCONFIG_NO_LEGACY_CONVERSION 1
 #endif

 /**
--- source/data/makedata.mak	2008-07-01 08:42:02.000000000 +0700
+++ source/data/makedata.mak	2008-08-21 16:33:43.000000000 +0700
@@ -219,9 +219,11 @@
 #
 #  Break iterator data files.
 #
+!IFDEF BRK_SOURCE
 BRK_FILES=$(ICUBRK)\$(BRK_SOURCE:.txt =.brk brkitr\)
 BRK_FILES=$(BRK_FILES:.txt=.brk)
 BRK_FILES=$(BRK_FILES:brkitr\ =brkitr\)
+!ENDIF

 !IFDEF BRK_CTD_SOURCE
 BRK_CTD_FILES = $(ICUBRK)\$(BRK_CTD_SOURCE:.txt =.ctd brkitr\)
@@ -350,7 +352,7 @@
 #				Building the common dll in $(ICUBLD_PKG) unconditionally copies it to $(DLL_OUTPUT) too.
 #
 #############################################################################
-ALL : GODATA "$(ICU_LIB_TARGET)" "$(TESTDATAOUT)\testdata.dat"
+ALL : GODATA "$(ICU_LIB_TARGET)"
	@echo All targets are up to date

 # Starting with ICU4C 3.4, the core Unicode properties files (uprops.icu, ucase.icu, ubidi.icu, unorm.icu)
@@ -395,11 +397,11 @@
	@echo Building icu data
	cd "$(ICUBLD_PKG)"
	"$(ICUP)\bin\pkgdata" $(COMMON_ICUDATA_ARGUMENTS) <<"$(ICUTMP)\icudata.lst"
-pnames.icu
-unames.icu
+#pnames.icu
+#unames.icu
 ucadata.icu
 invuca.icu
-uidna.spp
+#uidna.spp
 cnvalias.icu
 $(CNV_FILES:.cnv =.cnv
 )
@@ -653,11 +655,13 @@
 # This used to depend on "$(ICUBLD_PKG)\uprops.icu" "$(ICUBLD_PKG)\ucase.icu" "$(ICUBLD_PKG)\ubidi.icu" "$(ICUBLD_PKG)\unorm.icu"
 # This data is now hard coded as a part of the library.
 # See Jitterbug 4497 for details.
-$(MISC_SOURCE) $(RB_FILES) $(COL_COL_FILES) $(RBNF_RES_FILES) $(BRK_RES_FILES) $(TRANSLIT_RES_FILES): {"$(ICUTOOLS)\genrb\$(CFG)"}genrb.exe "$(ICUBLD_PKG)\ucadata.icu"
+$(UCM_SOURCE) $(MISC_SOURCE) $(RB_FILES) $(COL_COL_FILES) $(RBNF_RES_FILES) $(BRK_RES_FILES) $(TRANSLIT_RES_FILES): {"$(ICUTOOLS)\genrb\$(CFG)"}genrb.exe "$(ICUBLD_PKG)\ucadata.icu"

 # This used to depend on "$(ICUBLD_PKG)\uprops.icu" "$(ICUBLD_PKG)\ucase.icu" "$(ICUBLD_PKG)\ubidi.icu" "$(ICUBLD_PKG)\unorm.icu"
 # This data is now hard coded as a part of the library.
 # See Jitterbug 4497 for details.
+!IFDEF BRK_SOURCE
 $(BRK_SOURCE) : "$(ICUBLD_PKG)\unames.icu" "$(ICUBLD_PKG)\pnames.icu"
 !ENDIF
+!ENDIF
